components/List/index.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react'
+import {StyleSheet, View, Text} from 'react-native'
+import PropTypes from 'prop-types'
+
+export default List = props => (
+    static propTypes = {
```

Comment:
this won't compile. proptypes are declared after the function for pure components